src/loop.ts

```diff
@@ -253,8 +259,12 @@ async function singleTransferLoopPartM2S( optsLoop: TLoopOptions, strLogPrefix:
                         imaState.chainProperties.sc.joAccount,
                         imaState.chainProperties.mn.strChainName,
                         imaState.chainProperties.sc.strChainName,
-                        imaState.chainProperties.mn.chainId.toString(),
-                        imaState.chainProperties.sc.chainId.toString(),
+                        imaState.chainProperties.mn.chainId
+                            ? imaState.chainProperties.mn.chainId.toString()
```

Comment:
Is it possible to have null in `imaState.chainProperties.mn.chainId`?

Reply:
`chainId: string | number`, so... checking it as `bool` means checking string not empty